MacPorts does not want build systems to look in Homebrew directories.
--- configure.seed.orig	2021-10-27 13:43:08.000000000 -0500
+++ configure.seed	2021-12-21 01:12:39.000000000 -0600
@@ -54,8 +54,6 @@
 if test "x$with_openssl" != xno; then
   OLD_CFLAGS="${CFLAGS}"
   OLD_LDFLAGS="${LDFLAGS}"
-  CFLAGS="${CFLAGS} -I/usr/local/opt/openssl@1.1/include"
-  LDFLAGS="${LDFLAGS} -L/usr/local/opt/openssl@1.1/lib/"
   AC_CHECK_LIB([crypto], [EVP_CIPHER_CTX_reset])
   if test "x$ac_cv_lib_crypto_EVP_CIPHER_CTX_reset" != xyes; then
      AC_MSG_RESULT(OpenSSL 1.1 not present)
